Dockerfile

```diff
@@ -40,14 +40,20 @@ RUN apt-get update && apt-get install -y \
 	createrepo \
 	curl \
 	dpkg-sig \
+	e2fslibs-dev \
 	gcc-mingw-w64 \
 	git \
 	iptables \
 	libapparmor-dev \
 	libcap-dev \
 	libsqlite3-dev \
+	liblzma-dev \
+	libxml2-dev \
+	libz-dev \
 	mercurial \
 	parallel \
+	parted \
```

Comment:
@LK4D4 just to clarify -- neither ext2fsprogs nor parted is needed for building; they are only required for make test to work.